blog/templates/blog/updateEntry.html

```diff
@@ -0,0 +1,10 @@
+{% extends "blog/blogbase.html" %}
+
+{% block blog_content %}
+    <h1>Update {{ entry.title }}</h1>
+    
+    <form> method="post">{% csrf_token %}
```

Comment:
this markup is invalid a good indication is the coloring on this code review.  This line should instead be:

```
<form method="post">{% csrf_token %}
```

Also the form should have another attribute called "action" which is the url that you're trying to post the form data to.  So that should be something along the lines of:

```
<form method="post" action="{% url 'my-url-name' ... %}">{% csrf_token %}
```